README.md

```diff
@@ -30,6 +30,7 @@
 | [COV-482][cov-482] | NCBI-MANIFEST | [bucket][ncbi-bucket] | scheduled|
 | [COV-465][cov-465] | NCBI | [bucket][ncbi-bucket] | scheduled|
 | [COV-532][cov-532] | COM-MOBILITY | [here](com-mobility) | scheduled|
+| [COV-1151][cov-1151] | CITYOFCHICAGO | [here](cityofchicago) | scheduled|
```

Comment:
the link doesn't work, it's broken for the row above this one too, it should be `[here][...]` not `[here](...)`

Reply:
I changed it and made a new commit